nixos/modules/services/networking/biboumi.nix

```diff
@@ -0,0 +1,169 @@
+{ config, lib, pkgs, options, ... }:
+with lib;
+let
+  cfg = config.services.biboumi;
+  inherit (config.environment) etc;
+  apparmor = config.security.apparmor.enable;
+  rootDir = "/run/biboumi/mnt-root";
+  stateDir = "/var/lib/biboumi";
+  settingsFile = pkgs.writeText "biboumi.cfg" (
+    generators.toKeyValue {
+      mkKeyValue = k: v:
+        if v == null then ""
+        else generators.mkKeyValueDefault {} "=" k v;
+    } cfg.settings);
+  need_CAP_NET_BIND_SERVICE = cfg.settings.identd_port != 0 && cfg.settings.identd_port < 1024;
+  mkDisableOption = name: mkEnableOption name // { default = true; example = false; };
```

Comment:
You only use this once. If you think this is useful I suggest a separate PR and add to our `lib`.

Reply:
Done here: https://github.com/NixOS/nixpkgs/pull/94983

Reply:
Since `openFirewall` no longer defaults to `true`, there is no dependency on https://github.com/NixOS/nixpkgs/pull/94983 anymore.